lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -68,7 +70,9 @@ <h3 class="hd hd-3">${_("Student-Generated Certificates")}</h3>
                 <input type="submit" class="btn-blue" id="enable-certificates-submit" value="${_('Enable Student-Generated Certificates')}"/>
             </form>
         % else:
-            <p>${_("You must successfully generate example certificates before you enable student-generated certificates.")}</p>
+            <p class="under-heading">
```

Comment:
Adds some space between the text and the `Enable Student-Generated Certificates` button. (screenshot included)